src/syscall.c

```diff
@@ -18,21 +18,27 @@
  * system call: name, number
  */
 /* clang-format off */
-#define SUPPORTED_SYSCALLS       \
-    _(close,            57)      \
-    _(lseek,            62)      \
-    _(read,             63)      \
-    _(write,            64)      \
-    _(fstat,            80)      \
-    _(exit,             93)      \
-    _(gettimeofday,     169)     \
-    _(brk,              214)     \
-    _(clock_gettime,    403)     \
-    _(open,             1024)    \
-    IIF(RV32_HAS(SDL))(          \
-        _(draw_frame,   0xBEEF)  \
-        _(setup_queue,  0xC0DE)  \
-        _(submit_queue, 0xFEED), \
+#define SUPPORTED_SYSCALLS           \
+    _(close,                57)      \
+    _(lseek,                62)      \
+    _(read,                 63)      \
+    _(write,                64)      \
+    _(fstat,                80)      \
+    _(exit,                 93)      \
+    _(gettimeofday,         169)     \
+    _(brk,                  214)     \
+    _(clock_gettime,        403)     \
+    _(open,                 1024)    \
+    IIF(RV32_HAS(SDL))(              \
+        _(play_sfx,         0xBABE)  \
```

Comment:
You prosed 6 system calls for audio/sound. i.e.,
* play_sfx
* play_music
* stop_music
* set_music_volume
* setup_sound
* shutdown_sound

Can you use fewer ones? We may discuss on #152 before code reviewing.